test/verify/check-custom

```diff
@@ -0,0 +1,264 @@
+        b.click("a:contains('Edit description')")
+        b.wait_present("#cmpsr-modal-edit-description")
+        b.set_input_text("#textInput-modal-markup", updated_description)
+        b.click("#cmpsr-modal-edit-description button:contains('Save')")
+        b.wait_not_present("#cmpsr-modal-edit-description")
+        b.wait_present("a:contains('{}')".format(updated_description))
+        # backend got updated as well
+        desc = m.execute("""
+            composer-cli blueprints show openssh-server | grep description | \
+            awk -F '"' '{print $2}'
+            """).rstrip()
+        self.assertEqual(desc, updated_description)
+
+        # collect code coverage result
+        self.check_coverage()
+
+    def testHostname(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/composer", authorized=True)
+        b.wait_present("#main")
+
+        b.click("#openssh-server-name")
+        with b.wait_timeout(300):
+            b.click("#blueprint-tabs-tab-customizations")
+        # set hostname to "openssh-server"
+        b.click("button[aria-describedby=Hostname-help2]")
+        b.set_input_text("input[aria-label=Hostname]", "hostname-openssh-server")
+        b.click(".form-control-pf-save")
+        b.wait_present("span:contains(hostname-openssh-server)")
+        hostname = m.execute("""
+            composer-cli blueprints show openssh-server | grep hostname | awk -F '"' '{print $2}'
+            """).rstrip()
+        b.wait_present("span:contains({})".format(hostname))
+
+        # update hostname
+        b.click(".form-control-pf-value")
+        b.set_input_text("input[aria-label=Hostname]", "update-openssh-server")
+        b.key_press("\r")
+        b.wait_present("span:contains(update-openssh-server)")
+        hostname = m.execute("""
+            composer-cli blueprints show openssh-server | grep hostname | awk -F '"' '{print $2}'
+            """).rstrip()
+        b.wait_present("span:contains({})".format(hostname))
+
+        # clicking x button will not update
+        b.click(".form-control-pf-value")
+        b.set_input_text("input[aria-label=Hostname]", "no-update-openssh-server")
+        b.click(".form-control-pf-cancel")
+        b.wait_present("span:contains(update-openssh-server)")
+        hostname = m.execute("""
+            composer-cli blueprints show openssh-server | grep hostname | awk -F '"' '{print $2}'
+            """).rstrip()
+        b.wait_present("span:contains({})".format(hostname))
+
+        # invalid character, save button is disabled
+        b.click(".form-control-pf-value")
+        b.set_input_text("input[aria-label=Hostname]", "?")
+        b.wait_attr(".form-control-pf-save", "disabled", "")
+
+        # collect code coverage result
+        self.check_coverage()
+
+    def testUserAccount(self):
+        b = self.browser
+        m = self.machine
+
+        # work around "keystroke sent to browser" != "react component accepts it"
```

Comment:
It would be great if you would open issue on c-composer and put here `#HACK: cockpit-composer may loose keystrokes, see #<issue_number>`